package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@babel/runtime": "7.2.0",
     "@material-ui/core": "^3.9.1",
+    "@material-ui/icons": "^3.0.2",
```

Comment:
Considering the main goal of this PR, is this worth committing here?

Reply:
Hmm, I'm not sure why the material-ui version updated.